sei_dbt/README.md

```diff
@@ -91,17 +89,3 @@ More resources on building database pipelines via `dbt` and `duckdb` can be foun
 * https://duckdb.org/docs/api/python/overview.html
 
 
-Copyright 2024 Province of British Columbia
```

Comment:
I thought you needed this footer for every readme?

Reply:
My understanding is that it is needed in the repository `README.md` only, although the guidance [here](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Open-Source-Development-Employee-Guide/Licenses.md) does not really address this question.

Reply:
Yeah that aligns with their [cheatsheet](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Gov-Org-HowTo/Cheatsheet.md) as well - I think we're all good to remove it.